src/jquery.plusmore.js

```diff
@@ -86,15 +119,16 @@
       formatList: function(hiddenItems) {
          var firstHiddenItem = hiddenItems.first();
          var moreItem = firstHiddenItem.clone();
-         moreItem.removeClass(this.hiddenClass).html(" <a class=\"" + this.moreItemsClass + "\" href=\"#\">+ " + hiddenItems.length + " more</a>");
+         moreItem.removeClass(this.hiddenClass).addClass(this.moreItemsClass).html(" <a href=\"#\">+ " + hiddenItems.length + " more</a>");
          moreItem.click({hiddenItems: hiddenItems, hiddenClass: this.hiddenClass}, this.showMore);
```

Comment:
Moved the class addition to the element at the same level as the other items, made for a more direct implementation.  Thoughts on this?